---
abstract: Multivariate mixed models (MMM) are generalized linear models with both fixed and random effect having multiple response variables.  MMM allow partitioning of total (phenotypic) (co)variances for multiple traits into (co)variances on hierarchically lower levels. We outline why ecologists and evolutionary biologists should be interested in such partitioning as well as the levels of analyses that arise when making inferences on multi-level covariance structures. We consider biological levels of interest to be genes (e.g. GWAS, genomic selection), genotypes (genetic (co)variances), individual or other subject (e.g. between-individual (co)variances) and phylogenetic taxa (phylogenetic (co)variances). All of these biological levels can be modelled in a MMM and we distinguish several demand levels of using MMM of increasing complexity. We present an overview of current open-access software implementations of MMM in the open software R with respect to these demand levels, and present example scripts aimed at getting started with MMM on all biological levels. We describe four freely available R packages for MMM, two using Bayesian (Monte Carlo and Hamiltonian Markov Chains) and two using a likelihood framework. Depending on the need of the analyst, there are a number of freely available R-based MMM implementations of relevance to the field of ecology and evolution.
authors:
- Jon Brommer
- admin
- Giovanny Covarrubias-Pazaran
date: "2019-12-16T00:00:00Z"
doi: ""
featured: false
image:
  caption: ''
  focal_point: ""
  preview_only: false
links:
- name: EcoEvoRxiv
  url: https://ecoevorxiv.org/hs38a/
projects: [Project2]
publication: ""
publication_short: ""
publication_types:
- "3"
publishDate: "2019-12-16T00:00:00Z"
title: 'Multivariate Mixed Models in Ecology and Evolutionary biology: Inferences and implementation in R'
---
